packages/admin-e2e-tests/src/pages/WcHomescreen.ts

```diff
@@ -30,7 +30,7 @@ export class WcHomescreen extends BasePage {
 		await page.waitForSelector(
 			'.woocommerce-task-card .woocommerce-task-list__item-title'
 		);
-		await waitForElementByText( 'p', 'Get ready to start selling' );
+		await waitForElementByText( 'span', 'Get ready to start selling' );
```

Comment:
Would '*' work as well?

Reply:
I didn't know that was possible, but it is :)